tests/tnt/Makefile

```diff
@@ -54,7 +63,7 @@ cluster-down:
 # run go tests, minus "-" signs before command allows failures, otherwise cluster-down stage won't run.
 gotest:
 	@echo "${GREEN}STAGE: RUN GOTESTS${NC}"
-	-go test -race -parallel=20 -coverpkg="../../" -coverprofile cover.out -timeout=90s
+	go test -race -parallel=20 -coverpkg="../../" -coverprofile cover.out -timeout=90s
```

Comment:
Why we are removing "-" sign?

Reply:
cause i hope to get error when  non_existing_command